src/components/Articles.jsx

```diff
@@ -15,30 +19,47 @@ const Articles = () => {
     });
   }, [topicSlug]);
 
+  const navigate = useNavigate();
+  const clickHandler = (articleId) => {
+    navigate(`/articles/${articleId}`);
+  };
   return (
-    <section name="Articles List">
-      <header className="App-header">
-        Latest Articles for {!topicSlug ? "ALL topics" : `'${topicSlug}'`}
-      </header>
-      {isLoading ? <p className="App-header">Loading Articles</p> : null}
-      <div id="articleListBox">
-        {articlesList.map((ele) => {
-          return (
-            <div key={"articleCard" + ele.article_id} className="articleCard">
-              <ul>
-                <li className="articleCardTitle">{ele.title}</li>
-                <li className="articleCardTopic">topic: {ele.topic}</li>
-                <li className="articleCardCreated">Date: {ele.created_at}</li>
-                <li className="articleCardVotes">votes: {ele.votes}</li>
-                <li className="articleCardButtonLi">
-                  <button className="articleCardViewButton">View News</button>
-                </li>
-              </ul>
-            </div>
-          );
-        })}
-      </div>
-    </section>
+    <>
+      <Topics
+        topicFilter={topicFilter}
+        setTopicFilter={setTopicFilter}
+        topicsList={topicsList}
+        setTopicsList={setTopicsList}
+      />
+      <section name="Articles List">
+        <header className="App-header">
+          Latest Articles for {!topicSlug ? "ALL topics" : `'${topicSlug}'`}
+        </header>
+        {isLoading ? <p className="App-header">Loading Articles</p> : null}
+        <div id="articleListBox">
+          {articlesList.map((ele) => {
+            return (
+              <div key={"articleCard" + ele.article_id} className="articleCard">
+                <ul>
+                  <li className="articleCardTitle">{ele.title}</li>
+                  <li className="articleCardTopic">topic: {ele.topic}</li>
+                  <li className="articleCardCreated">Date: {ele.created_at}</li>
+                  <li className="articleCardVotes">votes: {ele.votes}</li>
+                  <li className="articleCardButtonLi">
+                    <button
+                      className="articleCardViewButton"
+                      onClick={() => clickHandler(ele.article_id)}
```

Comment:
The View News button doesn't seem quite right here. Thinking about a mobile screen, it would make sense for the whole card to be clickable.